app/httprequesthandler.py

```diff
@@ -23,19 +23,20 @@ def __init__(self, port):
 		self.acceptsock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
 		self.acceptsock.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
 		self.acceptsock.bind(("0.0.0.0", port))
-		self.acceptsock.listen(10)
 
 		self.broadcast = Broadcaster._instance
 		self.status = Status._instance
 
-		self.kill = False
-
 		self.acceptThread = threading.Thread(target = self.acceptClients)
 		self.acceptThread.daemon = True
 
 	def start(self):
+		self.acceptsock.listen(10)
 		self.acceptThread.start()
 
+	def stop(self):
+		self.acceptsock.close()

```

Comment:
Will this work correctly when we're listening for clients on line 103 in a different thread but with the same socket? If this does properly close the port on Windows and Linux then this is much cleaner. The reason it used the kill flag and the `quitsock` is to avoid acting on the socket from two different threads. Perhaps this works fine though, I haven't tested.